pkg/runtime/unstructured.go

```diff
@@ -26,12 +26,10 @@ import (
 // UnstructuredJSONScheme is capable of converting JSON data into the Unstructured
 // type, which can be used for generic access to objects without a predefined scheme.
 // TODO: move into serializer/json.
-var UnstructuredJSONScheme Decoder = unstructuredJSONScheme{}
+var UnstructuredJSONScheme Codec = unstructuredJSONScheme{}
```

Comment:
Also, "Scheme" is a confusing part of the name since we have a scheme and this is not much like it.